core/src/main/kotlin/com/inner/circle/core/service/UserSignUpService.kt

```diff
@@ -0,0 +1,19 @@
+package com.inner.circle.core.service
+
+import com.inner.circle.core.usecase.UserSignUpUseCase
+import com.inner.circle.infra.port.AccountSavePort
+import org.springframework.stereotype.Service
+
+@Service
+class UserSignUpService(
+    private val accountSavePort: AccountSavePort
+) : UserSignUpUseCase {
+    override fun signUp(account: UserSignUpUseCase.UserSignUpInfo) {
+        accountSavePort.saveAccount(
```

Comment:
```suggestion
        accountSavePort.save(
```

accountSavePort니 save라고 해도 충분하지 않을까요!?